source/_integrations/tado.markdown

```diff
@@ -178,13 +178,12 @@ automation:
         - sensor.temp_sensor_room
         - sensor.tado_temperature
     
-    # Check if the room temp is more than 0.5 higher than the tado thermostat reading
-    condition:
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading condition. The sensors default to room temperature (20) when the reading is in error:
```

Comment:
You've still removed `condition:` on this one, even in the last update.

Please restore that to make the template work.